components/dialog/dialog-mixin.js

```diff
@@ -0,0 +1,85 @@
+export const DialogMixin = superclass => class extends superclass {
+
+	static get properties() {
+		return {
+			opened: { type: Boolean, reflect: true },
+			title: { type: String },
+			width: { type: Number },
+			_state: { type: String, reflect: true }
+		};
+	}
+
+	constructor() {
+		super();
+		this.opened = false;
```

Comment:
Default value for `width`?

Reply:
I am a little torn where the `width` attribute should live.  Currently I have it defined in the mixin since I think the sizing logic will live in the mixin as well, but it is initialized in the constructor of `d2l-dialog` since the 600 is specific to it.  Confirmation dialog though I think is different.

Reply:
I've moved the `width` property to `d2l-dialog` since it's specific to it, while `d2l-dialog-confirm` relies on `max-width`.